stream-chat-android-client/src/main/java/io/getstream/chat/android/client/ChatClient.kt

```diff
@@ -269,7 +269,7 @@ internal constructor(
      * @see [Plugin]
      */
     @PublishedApi
-    internal var plugins: List<Plugin> = emptyList()
+    internal lateinit var plugins: List<Plugin>
```

Comment:
What if somebody calls an SDK method without setting the user? Will the error be better than not initialized property like with the line above?

Reply:
I would say that an error is better than an inconsistent state. But I think we should write a better message than "property not initialized"

Reply:
I checked, and when I call `createChannel` on the login screen the app crashes.
```
ChatClient.instance().createChannel("abc", "abc", emptyList(), emptyMap()).enqueue()
```

Reply:
Ok, let's keep with the initial state.